<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetCountInfoDlg</class>
 <widget class="BaseDlg" name="ResetCountInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">ResetCountInfoDlg</string>
  </property>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>293</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>293</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>部件名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sn_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>部件序列号</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>ResetCountInfoDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
